docs/malta.config.json

```diff
@@ -50,7 +51,7 @@
 				["Roblox", "/providers/roblox"],
 				["Salesforce", "/providers/salesforce"],
 				["Shikimori", "/providers/shikimori"],
-				["Slack", "/providers/slack"],
+				["Slack", "/providers/slack-openid"],
```

Comment:
This doesn't seem to point to the right page, the docs 404 (https://arcticjs.dev/providers/slack-openid)